bridges/opentracing/lib/opentelemetry/bridge/opentracing.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'opentracing'
+require 'opentelemetry'
+
+module OpenTelemetry
+  module Bridge
+    # OpenTracing Bridge provides a means of converting
+    # from OpenTelemetry to OpenTracing objects
+    OT = OpenTracing
+    module OpenTracing
+      FORMAT_TEXT_MAP = OT::FORMAT_TEXT_MAP
+      FORMAT_RACK = OT::FORMAT_RACK
```

Comment:
Do you need to define these constants? Clients should just use the same constants from the `::OpenTracing` namespace.

Reply:
Unneeded and removed